ts-packages/react-components/src/components/client-address/ClientAddress.tsx

```diff
@@ -1,8 +1,6 @@
-import React, { FC, useContext } from 'react';
+import React, { FC } from 'react';
```

Comment:
react `FC` is discouraged https://github.com/facebook/create-react-app/pull/8177

Reply:
Fair enough but we're using it extensively throughout the repo in all the TS apps. I think if we were to remove `FC` type we'd need to do it in one big PR for all apps and not this PR. Maybe we can talk to Mark about it some time.

Reply:
My point is not to refactor the whole code base here, but since we know that it's a bad practice and that we'll eventually end up by a refactor, why do not avoid using `FC` in new code?

Reply:
Yes I see. Small steps :) I'll refactor this file.